### ISSUE TITLE
Phoscon app deconz serial device connection

### ISSUE THREAD

**Issue by b23prodtm** [b23prodtm/HuewizPi#12]
As of deconz container settings request docker must run into su mode with privileged: true

**Comment by b23prodtm** [comment]
related to https://github.com/deconz-community/deconz-docker/issues/151
```
 failed to reconnect to network try=10
 raspbee-deconz-hue  14:18:14:988 dev /dev/ttyAMA0
 raspbee-deconz-hue  14:18:14:988 dev /dev/ttyS0
 raspbee-deconz-hue  14:18:14:988 COM: /dev/ttyS0 / serialno: , RaspBee
 raspbee-deconz-hue  14:18:15:919 Skip idle timer callback, too early: elapsed 916 msec
 raspbee-deconz-hue  14:18:16:419 reconnect network failed, try later
 raspbee-deconz-hue  14:18:16:919 start reconnect to network
 raspbee-deconz-hue  14:18:18:271 device state timeout (handled)
 raspbee-deconz-hue  14:18:18:351 void zmMaster::handleStateIdle(zmMaster::
```

**Comment by b23prodtm** [comment]
Fix #13
